bitnami/elasticsearch/templates/master-deploy.yaml

```diff
@@ -56,13 +56,14 @@ spec:
         - name: {{ . }}
       {{- end}}
       {{- end }}
-      initContainers:
-      - name: sysctl
-        image: {{ template "sysctl.image" . }}
-        imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
-        command: ["sysctl", "-w", "vm.max_map_count=262144"]
-        securityContext:
-          privileged: true
+      ## Image that performs the sysctl operation to modify Kernel settings (needed sometimes to avoid boot errors)
```

Comment:
I wouldn't comment it. I would put it as an option in the values

Reply:
+1